test/crowdsale/ZxcCrowdsale.test.js

```diff
@@ -40,11 +41,15 @@ contract('crowdsale/ZxcCrowdsale', (accounts) => {
   const buyerTwo = accounts[5];
   const _tester = accounts[6];  // tester should never be the default account!
   const xcertTokenOwner = accounts[7];
+  const buyerThree = accounts[8];
 
   let token;
   let xcertToken;
   let crowdsale;
 
+  const config = [web3Util.padLeft(web3Util.numberToHex(0), 64)];
```

Comment:
I think we need a helper function in utils which is taking care of this (not a fix requested in this PR tho)

Reply:
Yeah a function like numberToBytes32 would be welcomed. 

@xpepermint Can we package something like this directly into chainspin?